packages/symantec_endpoint_security/data_stream/incident/fields/fields.yml

```
@@ -0,0 +1,90 @@
+- name: ses.incident
+  type: group
+  fields:
+    - name: category
+      type: keyword
+    - name: category_id
+      type: keyword
+      description: Event type category.
+    - name: conclusion
+      type: keyword
+    - name: created
+      type: date
+      description: The creation time of the incident in ISO 8601 format.
+    - name: customer_uid
+      type: keyword
+      description: Customer id.
```

```suggestion
      description: Customer ID.
```
(`s/id/ID/gi` please)